docs/docs/contribute/docs/source-file-structure.md

```diff
@@ -13,6 +13,102 @@ This page explains how the documentation source files are organized.
   You may find small discrepancies between
   what is documented here and what is currently implemented.
 
+## Coding the documentation
+
+Keptn documentation is written using the Markdown language,
+with each page written in a separate file.
+Markdown supports many variants and the build tools we use
+impose a few special requirements that are discussed here.
+
+### Top of source requirements
+
+The top of each documentation source file must include:
+
+- A comments block
+- A level 1 header with the title of the page.
+  This must be preceded and followed by a single blank line.
+
+For example:
+
+```markdown
+---
+comments: true
+---
+
+# Uninstall
+
+```
+
+### Nested list format
+
+Traditional markdown coding has sub-items in a list
+lined up with the text of the item above.
+MkDocs, however, requires that:
+
+- Subitems be indented two additional spaces
+- Subsequent paragraphs for this sub-item
+  must be indented an additional two spaces
+  in order to line up with the text of the first paragraph.
+- Code blocks within the list must also be indented two spaces.
+
+Alas, `make markdownlint-fix` does not currently understand
+the formatting required for MkDocs
+and so will "correct" the indentation
+and you will end up with all list items
+rendered as left-justified
+and the numbers for numbered lists do not increment
+so all numbered items begin with "1.".
+Consequently, you must disable that particular markdownlint check
+at the beginning of the list
+and then re-enable it at the end of the list.
+
+- Disable markdownlint check for list indentation:
+
+    ```markdown
+    <!-- markdownlint-disable MD007 -->`
+    ```
+
+- Re-enable markdownlint check for list indentation:
+
+    ```markdown
+    <!-- markdownlint-disable MD007 -->`
```

Comment:
I think you made a typo here 
it should be `<!-- markdownlint-enable MD007 -->`

Reply:
resolved